requirements/app.in

```diff
@@ -61,8 +62,7 @@ uniplot>=0.7.0
 # >=2.5.2: https://github.com/marshmallow-code/flask-marshmallow/issues/262
 Flask-SQLAlchemy>=2.5.2
 # flask should be after all the flask plugins, because setup might find they ARE flask
-# <2.3: https://github.com/FlexMeasures/flexmeasures/issues/754
-flask>=1.0, <2.3
+flask>=1.0
```

Comment:
Hm, [Flask3 wants to install Werkzeug 3](https://github.com/pallets/flask/blob/main/pyproject.toml#L23), which we rule out (see line below). Would be interesting to see what combination will come out.

Is `make upgrade-deps` complaining?

Reply:
I see you have now ruled out Flask 2.3 (`<2.3`). This issue is about being able to move to 2.3.
Maybe you meant `<3` or `<=2.3`?

I'm assuming that was in response to this question of mine, although it's not answered or resolved.

Reply:
I've changed the line to `flask < 3.0`. I've also made a branch that upgrades the dependencies, flask is not moving to a higher version because of other dependencies however.